cmd/cmd.go

```diff
@@ -172,3 +178,36 @@ var burn = &cobra.Command{
 		fmt.Printf("Transaction: %s\n", tx.TxID)
 	},
 }
+
+var mine = &cobra.Command{
+	Use:   "mine",
+	Short: "Starts the PegNet mining daemon",
+	Long: "Starts the PegNet mining daemon. Takes number of miners to use as an optional " +
+		"argument, if not specified it will use the default settings from " +
+		"the configuration file",
+	Args:    cobra.MaximumNArgs(1),
+	Example: "pegnet mine 20",
+	Run: func(cmd *cobra.Command, args []string) {
+		Miners = ArgParseMining(args)
+
+		// Initalize hashtable bytemap
+		opr.LX.Init(0xfafaececfafaecec, 25, 256, 5)
```

Comment:
we're going with 30 bits, not 25

Reply:
Yeah this should really be a global value.

The grading_tests LX.init should also call the current hash block for correctness also in hindsight.

I won't be able to fix this until tomorrow, if someone can modify the PR am entirely on board.